src/App.tsx

```diff
@@ -10,33 +11,21 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setTabId] = useState('tab-1');
+  const selectedTab = tabs.find(tab => (tab.id === selectedTabId)) || tabs[0];
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${selectedTab.title}`}
       </h1>
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+      <Tabs
+        selectedTab={selectedTab}
```

Comment:
You don't need to pass this prop, because you could get access to the selected tab, because you have `tabs` and `selectedTabId`.
This line is redundund.